### PR TITLE
Use route function everywhere

### PR DESCRIPTION
**Pull request by asbiin**

### CODE REVIEW

**Comment by asbiin** [comment]
@djaiss every references to url are now handle by `route` function (except vor Vuejs or js urls ...). Named route in `web.php` are now grouped with [Route::name](https://laravel.com/docs/5.6/routing#route-group-name-prefixes) function.

File: routes/web.php [hunk 1]
```diff
@@ -80,167 +78,197 @@
...
-        // accidentely f**ked up the reminders table by messing up the contact ids
-        // and now the only way to delete those reminders is to bypass the ReminderRequest
-        // by creating a new route.
-        Route::delete('/people/{contact}/reminders/{rmd}', 'Contacts\\RemindersController@destroy')->name('.reminders.delete');
+        Route::name('reminders.')->group(function () {
+            Route::get('/people/{contact}/reminders/add', 'Contacts\\RemindersController@create')->name('add');
+            Route::post('/people/{contact}/reminders/store', 'Contacts\\RemindersController@store')->name('store');
+            Route::get('/people/{contact}/reminders/{reminder}/edit', 'Contacts\\RemindersController@edit')->name('edit');
+            Route::put('/people/{contact}/reminders/{reminder}', 'Contacts\\RemindersController@update')->name('update');
+            Route::delete('/people/{contact}/reminders/{reminder}', 'Contacts\\RemindersController@destroy')->name('delete');
```
Comment (asbiin) [inline routes/web.php]: @djaiss Is the problem you described always here (Special route to delete reminders) ? I replaced the route, and all is OK.

Comment (djaiss) [reply routes/web.php]: @asbiin really? Well if it works now, awesome.

**Comment by djaiss** [comment]
@asbiin is there a main benefit for this change? I do remember on Heroku having a problem with named routes for an unknown reason.

**Comment by asbiin** [comment]
@djaiss The (major) benefit for this change is ... clean code, beautiful aligned code. :D

**Comment by djaiss** [comment]
@asbiin alright! If it works, let's do this.

**Pull request merged**